config.toml

```diff
@@ -99,8 +101,24 @@ trusting_period = '14days'
 # which have changed between two blocks.
 # Default: { numerator = '1', denominator = '3' }, ie. 1/3.
 trust_threshold = { numerator = '1', denominator = '3' }
-[chains.filters]
-channels = [['transfer', 'channel-0']]
+
+# Channel filter specification. Default: no filter
+# This whole section can be deleted and then all filtering is disabled for this chain.
+# This section is ignored if the global 'filter' option is set to 'false'.
+# A channel filter has two fields:
+# 1. `policy` - Channel filter policy. One out of two types of policies are supported:
+#       - 'allow': permit relaying _only on_ the port/channel id in the list below,
+#       - 'deny': permit relaying on any channel _except for_ the list below.
+# 2. `list` - List of ports and channel identifiers.
+#
+# Example configuration of a channel filter, denying 'channel-0' on port 'transfer':
+#
+# [chains.channel_filter]
+# policy = 'deny'
+# list = [
+#   ['transfer', 'channel-0'],
+# ]
+
```

Comment:
Since the filter is for packets only it should be something like:
```suggestion
# [chains.packet_filter]
# policy = 'deny'
# channels = [
#   ['transfer', 'channel-0'],
# ]

```
I liked it a bit more the way it was before were all filters are grouped under `filters` section but it's ok to leave it like this for now. Should also probably revise at some point the `strategy`: `all` vs `packets` that can be expressed now with a `deny` in `chains.channel_filter` (assuming we change existing `chains.channel_filter` to `chains.packet_filter`).

Reply:
I also noticed a corner case: https://github.com/informalsystems/ibc-rs/discussions/1092#discussioncomment-973814

Reply:
fixed in 5978d01